src/test/java/hudson/plugins/timestamper/io/TimestampsWriterTest.java

```diff
@@ -161,7 +159,7 @@ public void testHashFile() throws Exception {
     byte[] expectedHash = MessageDigest.getInstance("SHA-1").digest(fileContents);
     assertThat(
         Files.toString(timestampsHashFile, Charsets.US_ASCII).trim(),
-        is(DatatypeConverter.printHexBinary(expectedHash).toLowerCase()));
+        is(bytesToHex(expectedHash).toLowerCase()));
```

Comment:
BTW always pass `Locale.ROOT` as the argument here, to avoid messing with Turkish developers. OTOH this is just test code and I suppose we should make the parent POM set the root locale in Surefire.